frontend/src/store/modules/shoots/getters.js

```diff
@@ -257,11 +297,44 @@ export default {
           break
         }
         default: {
-          items = orderBy(items, [item => getSortVal(rootGetters, item, sortBy), 'metadata.name'], [sortOrder, 'asc'])
+          itemsToSort = orderBy(itemsToSort, [item => getSortVal(rootGetters, item, sortBy), 'metadata.name'], [sortOrder, 'asc'])
         }
       }
-      return items
+
+      sortedUIDsAtFreeze = map(itemsToSort, ({ metadata }) => {
+        return metadata.uid
+      })
+      sortByArrAtFreeze = sortByArr
+      sortDescArrAtFreeze = sortDescArr
```

Comment:
I incorporated the feedback.
- The variables are now set outside of the sort function
- Copy the array values and compare the arrays instead of the pointers (to prevent this logic to break if vuetify changes the internal logic)